connector/signaltometricsconnector/README.md

```diff
@@ -20,6 +20,7 @@ logs, or metrics).
 | traces | metrics | [alpha] |
 | logs | metrics | [alpha] |
 | metrics | metrics | [alpha] |
+| profiles | metrics | [alpha] |
```

Comment:
Shall we add some examples to illustrate how the component can be used with this signal?

Reply:
I think we don't provide a reference to any example for any signal types as of now. The `testdata` directory has examples for all signal types, maybe we can add a new section for examples and refer to that directory?

Reply:
I was mostly thinking of sth basic, extending the [existing ones](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/connector/signaltometricsconnector/README.md?plain=1#L41-L55) so as to have this signal mentioned there as well. I'd be fine with linking to the other directory too, but this can be done outside of this PR.